applications/ConvectionDiffusionApplication/custom_elements/symbolic_d_convection_diffusion_explicit.cpp

```diff
@@ -0,0 +1,1783 @@
+//                   Kratos default license: kratos/license.txt
+//
+//  Main author:     Riccardo Tosi
+//
+
+#include "symbolic_d_convection_diffusion_explicit.h"
+
+namespace Kratos
+{
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+template< unsigned int TDim, unsigned int TNumNodes >
+SymbolicDConvectionDiffusionExplicit<TDim,TNumNodes>::SymbolicDConvectionDiffusionExplicit(
+    IndexType NewId,
+    GeometryType::Pointer pGeometry)
+    : SymbolicQSConvectionDiffusionExplicit<TDim,TNumNodes>(NewId, pGeometry) {}
+
+/***********************************************************************************/
+
+template< unsigned int TDim, unsigned int TNumNodes >
+SymbolicDConvectionDiffusionExplicit<TDim,TNumNodes>::SymbolicDConvectionDiffusionExplicit(
+    IndexType NewId,
+    GeometryType::Pointer pGeometry,
+    Properties::Pointer pProperties)
+    : SymbolicQSConvectionDiffusionExplicit<TDim,TNumNodes>(NewId, pGeometry, pProperties) {}
+
+/***********************************************************************************/
+
+template< unsigned int TDim, unsigned int TNumNodes >
+SymbolicDConvectionDiffusionExplicit<TDim,TNumNodes>::~SymbolicDConvectionDiffusionExplicit() {}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+template< unsigned int TDim, unsigned int TNumNodes >
+Element::Pointer SymbolicDConvectionDiffusionExplicit<TDim,TNumNodes>::Create(
+    IndexType NewId,
+    NodesArrayType const& ThisNodes,
+    Properties::Pointer pProperties) const
+{
+    return Kratos::make_intrusive<SymbolicDConvectionDiffusionExplicit>(NewId, this->GetGeometry().Create(ThisNodes), pProperties);
+}
+
+/***********************************************************************************/
+
+template< unsigned int TDim, unsigned int TNumNodes >
+Element::Pointer SymbolicDConvectionDiffusionExplicit<TDim,TNumNodes>::Create(
+    IndexType NewId,
+    GeometryType::Pointer pGeom,
+    Properties::Pointer pProperties) const
+{
+    return Kratos::make_intrusive<SymbolicDConvectionDiffusionExplicit>(NewId, pGeom, pProperties);
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+template< unsigned int TDim, unsigned int TNumNodes >
+void SymbolicDConvectionDiffusionExplicit<TDim,TNumNodes>::CalculateLocalSystem(
+    MatrixType& rLeftHandSideMatrix,
+    VectorType& rRightHandSideVector,
+    const ProcessInfo& rCurrentProcessInfo)
+{
+    KRATOS_TRY;
+
+    const auto& r_geometry = this->GetGeometry();
+    const unsigned int local_size = r_geometry.size();
+
+    // Resize and intialize output
+    if (rLeftHandSideMatrix.size1() != local_size)
```

Comment:
```suggestion
    if (rLeftHandSideMatrix.size1() != local_size || rLeftHandSideMatrix.size2() != local_size)
```

Reply:
Removing the `CalculateLocalSystem` method, and using `BoundedVector`, this change is no longer necessary. I leave to you the "resolution" of the conversation, in case you have further comments.